#include "qmk_midi.h" always results in compile error[Bug]

##  Describe the Bug

I've spent the past few days working on a keymap for the BDN9 that includes midi functionality on separate layers. Everything works great, but I wanted to include midi sends for the encoders like

```
        if (clockwise) {
            midi_send_cc(&midi_device, 0, 0x14, 1);
        } else {
            midi_send_cc(&midi_device, 0, 0x15, 1);
```

However, in order to do that I need to put #include "qmk_midi.h" within the keymap and no matter what I do it always results in the same error if qmk_midi.h is included. Here's the error

```
tmk_core/protocol/midi/qmk_midi.h:7:36: error: unknown type name 'MIDI_EventPacket_t'
 void              send_midi_packet(MIDI_EventPacket_t* event);
                                    ^~~~~~~~~~~~~~~~~~
tmk_core/protocol/midi/qmk_midi.h:8:36: error: unknown type name 'MIDI_EventPacket_t'
 bool              recv_midi_packet(MIDI_EventPacket_t* const event);

```

I've gotten pretty obsesive about it because I can't for the life of me figure it out. I have midi enabled in the rules and have # define MIDI_ADVANCED in the config. I'd really appreciate if anyone has any suggestion. Thanks

## System Information

 - Keyboard:  
   BDN9 REV 1 (Pro Micro)
 - Operating system:
    Windows 10
 - AVR GCC version: 
   8.3.0
 - ARM GCC version: 
    8.3.1
 - QMK Firmware version:
    0.9.10

--- a/tmk_core/protocol/midi/qmk_midi.h
+++ b/tmk_core/protocol/midi/qmk_midi.h
@@ -2,6 +2,7 @@
 
 #ifdef MIDI_ENABLE
 #    include "midi.h"
+#    include <LUFA/Drivers/USB/USB.h>
 extern MidiDevice midi_device;
 void              setup_midi(void);
 void              send_midi_packet(MIDI_EventPacket_t* event);

